src/web/src/components/Header.vue

```diff
@@ -85,6 +85,13 @@
           </b-modal>
         </div>
       </b-navbar-nav>
+      <b-form-checkbox
+        class="dark-mode-switch"
+        v-model="darkModeEnabled"
```

Comment:
other option instead of using `v-model`, you can use `:value=<initial value>` and `@input=<callback when value changes>`. here's the official docs: https://vuex.vuejs.org/guide/mutations.html#using-constants-for-mutation-types. using `v-model` works but adds an extra step in the process